docker/Dockerfile

```diff
@@ -11,7 +11,8 @@ ENV ZOO_CONF_DIR=/conf \
     ZOO_AUTOPURGE_SNAPRETAINCOUNT=3 \
     ZOO_MAX_CLIENT_CNXNS=60 \
     ZOO_STANDALONE_ENABLED=true \
-    ZOO_ADMINSERVER_ENABLED=true
+    ZOO_ADMINSERVER_ENABLED=true \
+    JVMFLAGS=-Dsun.net.inetaddr.ttl=0
```

Comment:
curious, why did we add this flag ?

Reply:
It's for the DNS resolution inside the container, sometimes the old IP address sticks to the process and it hangs the system.

Aren suggested it.